tools/alpine/Makefile

```diff
@@ -2,22 +2,24 @@
 
 ORG?=linuxkit
 IMAGE=alpine
-BASE=alpine:3.6
+
+BASE := $(shell bash -c 'source detect-arch-var && echo $${BASE}')
```

Comment:
very good suggestion. I will drop the 'detect-arch-var' file and transfer its operation into Makefile, so I promise 'bash' will not show again ;-)